src/pages/namespace/Explorer/components/ArrayInput.tsx

```diff
@@ -0,0 +1,106 @@
+import { FC, useState } from "react";
```

Comment:
This file is located under `pages/Explorer/etc`, but isn't it generic enough to live in the general (layout) components folder?

(or should we think about a common location for reusable form elements -- in case we don't have this yet?)

This probably applies to other components below as well.

Reply:
I think I just tend to place components to the closest shared root. But I can move it if you think this makes more sense. I would then  move the other component in the same folder as well

Reply:
I'm not familiar enough with these components to be 100% certain that they are generic enough to be used outside of this context. If they are, I think it makes sense to move them.

Reply:
I will do that then.